packages/block-library/src/cover/edit.js

```diff
@@ -316,6 +320,76 @@ function CoverEdit( {
 		allowedBlocks,
 		templateLock,
 	} = attributes;
+
+	const coverImage = useRef( {
```

Comment:
Can you clarify why the ref is needed (instead of just using the attribute)?

Reply:
The attribute is replaced by the featured image URL. If one toggles on and off the featured image control in the toolbar but has had a cover previously set this helps the block remember that between re-renders.